### ISSUE TITLE
Add admin UI for LB direct add button

### ISSUE THREAD

**Issue by pyrello** [uiowa/uiowa01#113]
The features that I would like to add, potentially:
* [ ] The option to choose which block type is the default and can be accessed with one click
* [ ] The option to choose the order of the custom block types listed in the button
* [ ] The ability to choose the number of items to show in the dropbutton, with the remaining options showing up under the main 'Add block' panel
* [ ] The option to select block types to not show up in the dropbutton, but still have them show up in the main 'Add block' panel

Dependent on #79

**Comment by pyrello** [comment]
This is now just a tracking issue for https://www.drupal.org/project/lb_direct_add/issues/3098555

**Comment by pyrello** [comment]
Closing this in favor of the issue on drupal.org.
